### ISSUE TITLE
Announce extensions list for Vortex predictably and/or automatically

### ISSUE THREAD

**Issue by madtisa** [Nexus-Mods/Vortex#14385]
**Is your feature request related to a problem? Please describe.**
Announcements happen in unpredictable times

**Describe the solution you'd like**
Automate announcement job (e.g. using github actions) and run it in scheduled manner

**Describe alternatives you've considered**
Asking users to download and install extension manually (from nexus) each time there is a bug

**Additional context**
This is the extensions list I'm talking about: https://github.com/Nexus-Mods/Vortex/blob/announcements/extensions_1_8.json

Once I published version of my extension that contained a bug, which is "announced" in hour. After that I fixed it and published hotfix to nexusmods. Unfortunately, next announcement had happened only after 3 day, during which there was 200 users that downloaded breaking version (because Vortex updates it automatically).
After second such case I decided to ask for help here.

**Comment by insomnious** [comment]
We are currently looking at the best solution so it can become an automated task

**Comment by insomnious** [comment]
Good point. I’ll leave the issue open and I’ll let you know. There are some manual elements to this pipeline, especially when adding new extensions and so I’m currently looking at how we can potentially do updating existing extensions fully automated as a first step.

**Comment by insomnious** [comment]
Yes, we’ve migrated to a new scheduled system that seems to be fine. We are
keeping an eye on it day to day but so far so good!
